lib/loadGltfUris.js

```diff
@@ -169,7 +168,7 @@ var jimpSupportedExtensions = ['.png', '.jpg', '.jpeg', '.bmp'];
 function generateJimpImage(object) {
     var pipelineExtras = object.extras._pipeline;
     var ext = pipelineExtras.extension;
-    if (jimpSupportedExtensions.indexOf(ext) === -1) {
+    if (supportedExtensions.test(ext)) {
```

Comment:
Silly mistake sorry. I appreciate the patience with me muddling my way through this first attempt at GitHub and my first attempt at contributing to a "real" project. I hope I haven't been to much of a disturbance in the operation of things for everyone. Thank you for allowing me to participate.